cdap-proto/src/test/java/co/cask/cdap/proto/id/EntityIdTest.java

```diff
@@ -115,8 +127,16 @@ public void testInvalidId() {
   @Test
   @Ignore
   public void testPrintToString() {
-    for (ElementId id : ids) {
-      System.out.println(id.toString());
+    for (EntityId id : ids) {
+      System.out.println(id.getEntity() + ": " + id.toString());
```

Comment:
seeing meaning manually? we should verify the output, no? You can set the ``PrintStream``s in ``System`` like [so](http://stackoverflow.com/a/1119559) and verify their contents?